modules/gatsby/src/gatsby-config/get-gatsby-config.js

```diff
@@ -339,6 +339,16 @@ module.exports = function getGatsbyConfig(config) {
         path: `${paddedConfig.DIR_NAME}/static/`
       }
     });
+    gatsbyConfig.plugins.push({
+      resolve: 'gatsby-source-filesystem',
+      options: {
+        name: 'images',
+        path: `${paddedConfig.DIR_NAME}/static/images`,
```

Comment:
- Should we `fs.stat` these directories before we add the plugins?
- Or `fs.stat` and warn if they don't exist?
- Or `fs.stat` and create them if they don't exist?